### PR TITLE
xdgmenumaker: update to 2.0

### PR DESCRIPTION
**Pull request by bugcrazy**
#### Testing the changes
- I tested the changes in this PR: **YES**

### CODE REVIEW

**Comment by bugcrazy** [comment]
This is xlint CI error message "authored by noreply email".

**Comment by paper42** [comment]
> This is xlint CI error message "authored by noreply email".

Correct, please fix that

**Comment by bugcrazy** [comment]
> > This is xlint CI error message "authored by noreply email".
> 
> Correct, please fix that

Done!

File: srcpkgs/xdgmenumaker/template [hunk 1]
```diff
@@ -10,7 +10,7 @@ maintainer="reback00 <reback00@protonmail.com>"
 license="GPL-3.0-or-later"
 homepage="https://github.com/gapan/xdgmenumaker"
 distfiles="https://github.com/gapan/${pkgname}/archive/${version}.tar.gz"
```
Comment (kruceter) [inline srcpkgs/xdgmenumaker/template:12]:
```suggestion
distfiles="https://github.com/gapan/xdgmenumaker/archive/${version}.tar.gz"
```

**Review by kruceter — APPROVED** [review]
Looks alright to me. 2.0 goes right after 1.6.

The log contains 7 commits, most of them drop python 2 support and replace deprecated functions. No change in dependencies occured.

**Comment by bugcrazy** [comment]
> Looks alright to me. 2.0 goes right after 1.6.
> 
> The log contains 7 commits, most of them drop python 2 support and replace deprecated functions. No change in dependencies took place.

Version 1.6 has obsolete functions, version 2.0 fixes it, this is the reason for the template update!

File: srcpkgs/xdgmenumaker/template [hunk 2]
```diff
@@ -1,16 +1,16 @@
 # Template file for 'xdgmenumaker'
 pkgname=xdgmenumaker
-version=1.6
-revision=2
+version=2.0
+revision=1
 build_style=gnu-makefile
 hostmakedepends="txt2tags"
 depends="python3-xdg python3-gobject gobject-introspection python3-Pillow"
```
Comment (paper42) [inline srcpkgs/xdgmenumaker/template:7]:
```suggestion
depends="python3-xdg python3-gobject gobject-introspection python3-Pillow gtk+3"
```

Comment (bugcrazy) [reply srcpkgs/xdgmenumaker/template:7]: It is not required dependency for build, the software does not use gtk+3. What is the reason for using gtk+3?

Comment (paper42) [reply srcpkgs/xdgmenumaker/template:7]: `$depends` is a list of runtime dependencies, not build dependencies. xdgmenumaker uses it [here](https://github.com/gapan/xdgmenumaker/blob/7dd8a63abd0de0151708339532c6a717b2741328/src/xdgmenumaker#L311) to figure out the icon path.

Comment (bugcrazy) [reply srcpkgs/xdgmenumaker/template]: Done!

**Pull request merged**